Source/WTF/wtf/posix/FileSystemPOSIX.cpp

```diff
@@ -177,22 +182,30 @@ std::optional<uint64_t> fileSize(PlatformFileHandle handle)
 
 std::optional<WallTime> fileCreationTime(const String& path)
 {
-#if OS(DARWIN) || OS(OPENBSD) || OS(NETBSD) || OS(FREEBSD)
+#if OS(LINUX) && HAVE(STATX) || OS(DARWIN) || OS(OPENBSD) || OS(NETBSD) || OS(FREEBSD)
```

Comment:
Don't we need parentheses? 
```suggestion
#if (OS(LINUX) && HAVE(STATX)) || OS(DARWIN) || OS(OPENBSD) || OS(NETBSD) || OS(FREEBSD)
```

Reply:
I don't think the parentheses are required, but they certainly won't hurt either.